client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -86,23 +86,33 @@ export default {
         cardRank = this.selectedCard.rank;
       }
       // Re-usable move-objects
+      const allMovesAreDisabled = !this.isPlayersTurn || this.frozenId === this.selectedCard.id;
```

Comment:
Longer term, I think this whole function needs to be refactored. It's a bit of a mess and it's making it hard to understand what is actually happening throughout the logic chains which I'm sure isn't helping your understanding of it all.

Reply:
It makes sense to me now, but im sure coming back to this or someone coming into it would certainly be a little confused :/